sympy/polys/matrices/sdm.py

```diff
@@ -75,28 +211,108 @@ def to_list(M):
         return ddm
 
     def to_ddm(M):
+        """
+        Convert a :py:class:`~.SDM` object to a :py:class:`~.DDM` object
+
+        Examples
+        ========
+
+        >>> from sympy.polys.matrices.sdm import SDM
+        >>> from sympy import QQ
+        >>> A = SDM({0:{1:QQ(2)}, 1:{}}, (2, 2), QQ)
+        >>> A.to_ddm()
+        [[0, 2], [0, 0]]
+
+        """
         return DDM(M.to_list(), M.shape, M.domain)
 
     def to_sdm(M):
         return M
 
     @classmethod
     def zeros(cls, shape, domain):
+        """
+
+        Returns a :py:class:`~.SDM` of size shape,
+        belonging to the specified domain
+
+        Examples
+        ========
+
+        >>> from sympy.polys.matrices.sdm import SDM
+        >>> from sympy import QQ
+        >>> A = SDM.zeros((2, 3), QQ)
+        >>> A
+        {}
```

Comment:
This would be easier to understand if you also show what the corresponding matrix would be.

Reply:
You could show what `A.to_Matrix()` gives here